src/index.js

```diff
@@ -3,35 +3,38 @@ import { processResults } from './junit-xml'
 import parseOptions from './parse-options'
 import 'core-js/shim'
 import Logger from './logger'
+import _ from 'lodash'
 
 export default function (options = {}, callback = function noop () {}) {
   let parsedOptions = parseOptions(options)
   let testAttempt = parsedOptions.testAttempt || 1
   let logger = new Logger(parsedOptions.color)
 
-  function rerunFailedTests () {
+  function rerunFailedTests (status, output) {
```

Comment:
What is `output` ? Write JsDOC or

Reply:
It is stdout and stdin from protractor's process.